depends/packages/freetype.mk

```diff
@@ -1,8 +1,8 @@
 package=freetype
-$(package)_version=2.7.1
-$(package)_download_path=https://download.savannah.gnu.org/releases/$(package)
-$(package)_file_name=$(package)-$($(package)_version).tar.bz2
-$(package)_sha256_hash=3a3bb2c4e15ffb433f2032f50a5b5a92558206822e22bfe8cbe339af4aa82f88
+$(package)_version=2.10.4
+$(package)_download_path=http://download.savannah.gnu.org/releases/$(package)
```

Comment:
No opinion on whether to use http vs https

Reply:
Indeed. Also, please update the docs as done in commit 83582dcf23c23326fcb648b436d51469e66db5d5

Reply:
No probs, i'll test with https and update the docs too.
Closing